### DISCUSSION TITLE
19.12.2023 - For, For-Loops, Nested For-Loops

### DISCUSSION THREAD

**Discussion by DGrillCoder** [grilldb97/legendary-space-zebra#16] [Show and tell]
Diese Woche haben uns in einer Vorlesung For, For-Loops, Nested For-Loops angeschaut.
Wir sollten uns an dieser Herausforderung üben:
![image](https://github.com/grilldb97/legendary-space-zebra/assets/152898829/7be7c3d0-0fb9-4ba3-bbde-2844232223a8)
Es war viel Testen, bis ich die linke Darstellung einigermaßen hinbekommen habe, aber leider habe ich die 'o' Darstellung nicht rechtsbündig hinbekommen. Das muss ich mir auf jeden Fall mal noch weiter anschauen und dran bleiben.

Des weiteren haben wir uns die turtle Libary in Python angeschaut. 
Damit kann man zeichnen. In Kombi mit For, For-Loops, Nested For-Loops war das eine spannende Sache
und man kann sich lange damit beschäftigen.

**Comment by SirPatschiii** [comment]
Kleiner Tipp zu der 'o'- Darstellung. Vielleicht hilft es dir, die Leerzeichen mit ggf. einer extra for-loop auszurechnen und damit die Formatierung richtig hinzubekommen.
Ansonsten hört sich das doch ganz gut an :D

**Comment by ArthurFleck35x** [comment]
Bleib weiter so am Ball, durch herumtesten lerne ich zum Beispiel sehr viel gelernt . Man braucht wirklich manchmal ein bisschen Zeit. Ist was gutes bei dir raus gekommen?

**Reply by DGrillCoder** [reply]
![image](https://github.com/grilldb97/legendary-space-zebra/assets/152898829/fd4863a5-b469-46b4-9bbf-3ea2795206f3)
so sieht mein Ergebnis aus. also hab ich noch bisschen was zu tun :D 

Ansonsten hab ich gestern einen Wecker gestartet zu programmieren. Mal schauen wie weit ich das bringen kann
